app/assets/stylesheets/restaurants.scss

```diff
@@ -1,3 +1,112 @@
-// Place all the styles related to the Restaurants controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin make-float-boxes(){
```

Comment:
Avoiding floats, absolutes and relatives is better if they're not necessary. They're stylesheet the belong to "the old world", today we can achieve most layouts using `flexbox` in CSS3 which is a much better practice. Although floats and absolutes are still needed in some cases, trying to avoid them would benefit the layout greatly in the long run.